lib/perl/Genome/Sys/Lock/NessyBackend.pm

```diff
@@ -178,6 +181,32 @@ sub min_timeout {
     return 5;
 }
 
+sub _translate_resource {
```

Comment:
This might work as well:

```
sub _translate_resource {
    my ($self, $resource) = @_;
    my $genome_lock_dir = Path::Class::Dir->new($ENV{GENOME_LOCK_DIR});
    my $resource_file = Path::Class::File->new($resource);
    my $resource = $genome_lock_dir->contains($resource_file)
                 ? $resource_file->relative($genome_lock_dir)->stringify
                 : $resource_file->stringify;
    return $resource;
}
```